### ISSUE TITLE
Incompatible with latest AnkiTools version?

### ISSUE THREAD

**Issue by dwks** [AurelienLourot/lsankidb#2]
Hello, I just tried to install this and ran it while anki was running in the background, but I get an import error. It looks like AnkiTools.tools just doesn't exist anymore?
```
$ lsankidb 
...
  File ".../lib/python3.7/site-packages/src/lsankidb.py", line 9, in <module>
    import AnkiTools.tools.read
ModuleNotFoundError: No module named 'AnkiTools.tools'
```
P.S. "import AnkiTools" works just fine.

**Comment by lourot** [comment]
Hi @dwks, indeed we're stuck on an older version of AnkiTools for now, see the reason [here](https://github.com/AurelienLourot/lsankidb/issues/1#issuecomment-461611802).

**Comment by dwks** [comment]
OK, installing
```
pip install AnkiTools==0.1.9
```
works. Not sure why I didn't get that version from requirements.txt; perhaps I already had a newer version installed. Thanks.

**Comment by dwks** [comment]
Actually, this is reproducible by issuing
```
$ python3 -m venv env
$ source env/bin/activate
$ pip install lsankidb
$ lsankidb
```
on Debian 10 (buster) with python 3.7.3. It pulls in AnkiTools 0.3.7. The version of lsankidb is 1.0.0.

**Comment by lourot** [comment]
Indeed, we should change [this line](https://github.com/AurelienLourot/lsankidb/blob/master/setup.py#L6) from:

```
      install_requires=['AnkiTools'],
```

to:

```
      install_requires=['AnkiTools==0.1.9'],
```
